package.json

```diff
@@ -66,7 +66,7 @@
     "netrc-parser": "^3.1.6",
     "open": "^8.2.1",
     "rimraf": "^3.0.2",
-    "typescript": "^4.0.5"
+    "typescript": "4.3.4"
```

Comment:
Because of oclif dependency. With newer TS you get an error during build. To resolve that I would have to migrate oclif to their newer package structure which breaks some of the types that we are using. I tried to do it yesterday but I didn't get far and it started to feel like refactoring. I plan to do it in separate PR because it will be a lot of changes.